.github/CODEOWNERS

```diff
@@ -58,7 +58,7 @@ Callout/  @joschect
 # ComboBox/
 CommandBar/   @micahgodbolt
 ContextualMenu/  @joschect
-DatePicker/   @johannao76 lorejoh12@gmail.com jolore@microsoft.com
+DatePicker/   @johannao76 lorejoh12@gmail.com jolore@microsoft.com kegorr@microsoft.com
```

Comment:
Why this change?

Reply:
I thought it would help to have a third person, I'll remove it!